xds/src/main/java/io/grpc/xds/internal/sds/XdsServerBuilder.java

```diff
@@ -40,7 +38,6 @@
  * A version of {@link ServerBuilder} to create xDS managed servers that will use SDS to set up SSL
  * with peers. Note, this is not ready to use yet.
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/6268")
```

Comment:
Same thing here. Until XdsCreds is implemented we would still treat this as an experimental API. Unless the fact that this is now in "internal" makes Experimental unnecessary but I think internal and experimental are 2 different things

Reply:
We can close #6268 and open another once we have to add `XdsServerBuilder` in `io.grpc.xds`.
`@Experimental` means the public API is not stable. `@Interanl` is not "public" API and it's inherently unstable.